src/ranged.cpp

```diff
@@ -435,6 +435,10 @@ double player::gun_engagement_range( const item& gun, engagement opts, int penal
         case engagement::absolute_max:
             return gun_engagement_range( gun, -1, penalty, 10, accuracy_grazing );
     }
+    // return sane value in event engagement is not handled. 
+    debugmsg( "gun_engagement_range: engagement value not handled!!"); 
```

Comment:
You don't need to mention the function name here: `debugmsg()` will do that for you.